edit.ascx

```diff
@@ -0,0 +1,129 @@
+                    </div>
+                </header>
+            </div>
+
+                <!-- Page content-->
+                <main class="skin-main content-wrap">
+               
+                
+                <div class="panes">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col col-12">
+                                <div class="pane content-pane" id="contentpane" runat="server" />
+                            </div>
+                        </div>
+                    
+                    </div>
+                </div>
+                
+                </main>
+                <footer class="skin-footer bg-blue-super-dark skew-top">
+                    <div class="container py-5">
+                        <div class="row gx-5">
+                            <div class="col">
+                                <div class="pane footer_pane_1 col" id="footer_pane_1" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_2 col" id="footer_pane_2" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_3 col" id="footer_pane_3" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_4 col" id="footer_pane_4" runat="server">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="pane footer_pane_full_width pb-0" id="footer_pane_full_width" runat="server">
+                    </div>
+                    <div class="footer-bottom">
+                        <div class="container pt-2">
+                            <div class="row">
+                            <!-- #include file="Includes/footer-bottom.ascx" -->
+                            </div>
+                        </div>
+                    </div>
+                </footer>
+                
+               
+            </div>
+ 
+        <!-- Bootstrap core JS-->
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
+        <!-- Core theme JS-->
+		<dnn:DnnJsInclude runat="server" ForceProvider="DnnFormBottomProvider" FilePath="vendors/theme/js/scripts.js" PathNameAlias="SkinPath" />
+        <!-- FontAwsome 4-->
+        <dnn:DnnCssInclude runat="server" FilePath="vendors/fontawesome-free-5.15.4-web/css/all.css" PathNameAlias="SkinPath" />
+        <dnn:DnnCssInclude runat="server" FilePath="vendors/fontawesome-free-5.15.4-web/css/v4-shims.min.css" PathNameAlias="SkinPath" />
```

Comment:
This also appears to not exist in this repo and should be loaded from npm package and managed from there for better maintainability.

Reply:
Hmm, I have not found a way to the v4-shims from npm. We need as some modules still use FA4.
If you know of a way to get that from NPM, please let me know. :-)

Reply:
For what it is worth I built my own "shim" that also console logs usages of the deprecated classes so one can navigate the site and know which content needs to be changed to fa5. It is available here https://github.com/eraware/Eraware.MinimalSkin/blob/develop/src/scripts/utilities/utils.ts and I am happy to help if anything is missing.

Reply:
@valadas I used the one I tested for now